### ISSUE TITLE
Training from scratch [Data format query]

### ISSUE THREAD

**Issue by sanchit-ahuja** [RicherMans/Datadriven-GPVAD#18]
Hi,
Thank you for your wonderful work with GPVAD. I am looking at training the student model from scratch for my dataset(s). My dataset is in the form of audio_signal (wav) and the region has been tagged within the audio sample. 
For example:
```[{'type': 'BACKGROUND NOISE', 'time-range': [3.041, 3.169]}, {'type': 'SPEECH', 'time-range': [5.208, 5.544]}, {'type': 'BACKGROUND NOISE', 'time-range': [4.339, 5.069]}]``` is a tagged audio. Can your data pipeline support training for such data formats? If not, what do you suggest I should do to find a work around this?
Thanks a lot!

**Comment by sanchit-ahuja** [comment]
@RicherMans Can you help with the data format? I also wanted to fine-tune the student network via the teacher labels and see how it performs. I am unable to decode how to go about the data format hurdle. Thanks a lot!

**Comment by RicherMans** [comment]
Hey sorry didn't get the notification :(.
No that pipeline does not support that format, since my goal in this work is to use weak-only labeled datasets and then just use (soft) predictions from a teacher to a student. 
You need to write that by yourself I guess.

**Comment by sanchit-ahuja** [comment]
Alright. Maybe splitting the audio files (at their respective intervals) with their labels and generating the dataset in your format is the way to go.

**Comment by RicherMans** [comment]
So I guess you might misunderstand what this repo is for or what the goal is:
It solves the partial issue of requiring any frame-level alignment provided by some external source, such as an HMM.
The goal of this work is exactly not to use data such as yours, since this sort of data can be just trained by any other VAD system.
GPVAD supports to use any arbitrary clip-level training without explicit supervision, which is its selling point.
However I already ran joint GPVAD training with additional frame-level labels and have seen an improvement in performance ( usually only for clean scenarios ).
So feel free to try that out!

**Comment by sanchit-ahuja** [comment]
I did try inferencing on the gpvad model and the performance of this model is better than VADC or any normally available VAD in the literature. I just realised that I do not have large amounts of data required to train a robust VAD system so I am planning to just finetune it on our available dataset. My data does consist of noisy audios and that is why your model is able to give me decent performance on inferencing only. So I thought maybe finetuning on your model would help me achieve better performance.

**Comment by RicherMans** [comment]
Yeah, I guess it would ofc improve performance, but just the current codebase does not support that stuff, since I didn't think people would even fine-tune these models :(.
But have a try! and let me know the results, I'm always happy to know that some stuff actually works in real-world scenarios :)

**Comment by sanchit-ahuja** [comment]
Sure. Thanks a lot for this!
